autopush/web/registration.py

```diff
@@ -133,6 +137,41 @@ def validate_data(self, data):
                                          errno=109,
                                          headers=request_pref_header)
 
+    @post_load
+    def handler_kwargs(self, data):
+        # not used
+        data.pop('auth')
+        router_type = data.pop('router_type')
+        data['rinfo'] = RouterInfo(
+            router=self.context['settings'].routers[router_type],
+            type_=router_type,
+            token=data.pop('router_token'),
+            data=data.pop('router_data')
+        )
+
+
+@attrs(slots=True)
+class RouterInfo(object):
+    """Bundle of Router registration information"""
+
+    router = attrib()  # type: Any
```

Comment:
It's really Any? I thought there were only a few valid sets of things it could be.